code/interfaces/CronTaskEditable.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * By implementing this interface you can define this task as editable
+ * and can implement needed logic who can edit it
+ *
+ */
+interface CronTaskEditable {
```

Comment:
OK - because of how class manifest works, we can't look for inherited implementors (ie: `ClassInfo::implementorsOf()` won't match `CronTaskEditable` implementors when looking for `CronTask` implementors...